src/contexts/UserDataContext.tsx

```diff
@@ -270,7 +331,6 @@ export const UserDataContextProvider = (props: {
                                     }
                                 }
                             }
-                            dispatch(setRecentTokens(result));
```

Comment:
This was setting the recentTokens in RTK, but those values were not being used anywhere.  It's possible this should be referencing `useRecentTokens` hook.  However, as is this code is dead.